Sources/wkhtmltopdf/Option.swift

```diff
@@ -0,0 +1,640 @@
+import Foundation
+
+/// Represents wkhtmltopdf options
+public enum Option: Sendable {
```

Comment:
I'm tempted to say "make this `Equatable`" (maybe even `Hashable`) but while the conformance isn't at all complicated, it'd still be a miserable slog to write, and it doesn't really need it.